packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -269,18 +269,18 @@ export default class TradeStore extends BaseStore {
             : // else requests new active_symbols
               await WS.authorized.activeSymbols();
         if (error) {
-            this.root_store.common.showError(localize('Trading is unavailable at this time.'));
+            this.root_store.common.showError({ message: localize('Trading is unavailable at this time.') });
             return;
         } else if (!active_symbols || !active_symbols.length) {
             if (this.root_store.client.landing_company_shortcode !== 'maltainvest') {
                 showUnavailableLocationError(this.root_store.common.showError);
                 return;
             } else if (this.root_store.client.landing_company_shortcode === 'maltainvest') {
                 showDigitalOptionsUnavailableError(this.root_store.common.showError, {
-                    title: localize(
+                    text: localize(
```

Comment:
I think we can share all these error messages in digital-options.js in shared as well?